lib/blocks.php

```diff
@@ -85,6 +63,87 @@ function gutenberg_reregister_core_block_types() {
 }
 add_action( 'init', 'gutenberg_reregister_core_block_types' );
 
+/**
+ * Complements the implementation of block type `core/social-icon`, whether it
+ * be provided by core or the plugin, with derived block types for each
+ * "service" (WordPress, Twitter, etc.) supported by Social Links.
+ *
+ * This ensures backwards compatibility for any users running the Gutenberg
+ * plugin who have used Social Links prior to their conversion to block
+ * variations.
+ *
+ * This shim is INTENTIONALLY left out of core, as Social Links haven't yet
+ * landed there.
+ *
+ * @see https://github.com/WordPress/gutenberg/pull/19887
+ */
+function gutenberg_register_legacy_social_link_blocks() {
+	$sites = array(
```

Comment:
It probably could be:
```suggestion
	$services = array(
```

but I see why it is called `$sites`. It's your call what to land.

Reply:
That works — done.